openpdf/src/main/java/com/lowagie/text/html/HtmlWriter.java

```diff
@@ -295,15 +356,15 @@ public boolean add(Element element) throws DocumentException {
                         MarkedObject mo = ((MarkedSection) element).getTitle();
                         if (mo != null) {
                             markup = mo.getMarkupAttributes();
-                            mo.process(this);
+                            process(this);
                         }
-                        ms.process(this);
+                        process(this);
```

Comment:
Here you are not calling the process-method of each object, but the process-method of this. This don't look right.